docs/integrations_sso.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Integration based on single sign on
+parent: Integrations
+has_children: false
+has_toc: false
+nav_order: 5
+---
+
+# Integration based on single sign on
+Use this option for accessing our services by logging in with a supported identity provider.
+
+**Pre-requisite:**
+- You should have an account with a federated identity provider
```

Comment:
You must have registered your application in your identity provider with optional application secret.
You have an application ID (client_id) from above step and the JWK URL.